tasks/main.yml

```diff
@@ -1,7 +1,6 @@
 ---
-# Set Rsyslog facts
-
-- block:
+- name: Set Rsyslog facts then include rsyslog role
+  block:
 
     - name: Set rsyslog_enabled
       set_fact:
```

Comment:
given that everything is in a block, could you set this in a variable on the block, instead of using set_fact? Or are you using this fact even outside the block?

Reply:
If we include_role "{{ role_path }}/roles/rsyslog" in the block, then can we say everything under roles/rsyslog is in the block?  If the answer is yes, then we could set them in a variable on the block.

Reply:
Should be the case, you are using this already in another PR, no?

Reply:
Well, these set_fact calls were untouched in https://github.com/linux-system-roles/logging/pull/87.  The use case of the variables here are quite different from the other cases.  The vars fixed in pr/87 are used in the included role once, immediately.  But the vars set in this block by set_fact are supposed to be global and used in many places some times in the nested way...